index.html

```diff
@@ -15,14 +15,24 @@
         <div class="navbar">
           <a href="#" class="logo">Patrick Maina</a>
           <nav class="side-nav">
+            
             <ul class="nav-items">
               <li class="item-1"><a class="item-name-1" href="#">Portfolio</a></li>
               <li class="item-2"><a class="item-name-2" href="#">About</a></li>
               <li class="item-3"><a class="item-name-3" href="#">Contact</a></li>
-              <li class="mail-icon"><a href="#"><img class="mail-bar" src="Union.svg" alt="Mail bar"></a></li></a></li>
+              <li class="mail-icon"><a href="#"><img class="mail-bar" src="Images/Union.svg" alt="Mail bar"></a></li></a></li>
             </ul>
```

Comment:
- [ ] For this project you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) design as much as possible. Your design is missing the right arrow icons in front of menu options and the background images. Please make sure to include them in your design and your design should match the Figma template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/174986580-21c62fcc-501a-4f65-8772-d1ecbd293231.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/174986515-168b9539-60a6-430f-bdde-cd609bfa220b.png)